pkg/queue/health/health_state.go

```diff
@@ -95,7 +97,7 @@ func (h *State) HealthHandler(prober func() bool) func(w http.ResponseWriter, r
 		}
 
 		switch {
-		case h.IsAlive():
+		case isNotAggressive && h.IsAlive():
```

Comment:
I kind of envisioned the change to be `!isAggressive` here. But I guess this works also :)

Reply:
will brainstorm a more descriptive name and update in future 🤔